moto/ec2/responses/launch_templates.py

```diff
@@ -0,0 +1,247 @@
+import six
+import uuid
+from moto.core.responses import BaseResponse
+from moto.ec2.models import OWNER_ID
+from moto.ec2.exceptions import FilterNotImplementedError
+from moto.ec2.utils import filters_from_querystring
+
+from xml.etree import ElementTree
+from xml.dom import minidom
+
+
+def xml_root(name):
```

Comment:
Curious why the XML parsing functions are needed. If I recall, most of the others don't have these.

Reply:
The other EC2 mocks that I looked at all use string templates which are just xml blobs waiting to be filled in. I thought generating the xml rather than filling in string templates was safer and, when it came to more complex objects, easier to read.

Reply:
Sounds good.

Only thing I would ask is to test for the lack of coverage here: https://coveralls.io/builds/25143402/source?filename=moto%2Fec2%2Fresponses%2Flaunch_templates.py#L27

Reply:
👍